src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The `calendar--start-day-sun` and `calendar--month-length-31` modifiers should be added programmatically based on the actual month data, rather than hardcoding them. This allows for more dynamic and reusable code.